### ISSUE TITLE
Wrong language being recorded

### ISSUE THREAD

**Issue by jamesone** [github/linguist#3286]
I have a PHP repo that has a heap of built css/sass & it's saying the branch is css. Can I tell linguist to ignore a particular folder using the `.gitattributes` file?

**Comment by Alhadis** [comment]
Erh, how did you know about the `.gitattributes` file without reading [this](https://github.com/github/linguist#using-gitattributes)...?

> Use the linguist-vendored attribute to vendor or un-vendor paths.
```
$ cat .gitattributes
special-vendored-path/* linguist-vendored
jquery.js linguist-vendored=false

**Comment by jamesone** [comment]
So if I wanted to un-vendor /path/to/css/ would I do something like:
`
path/to/css/* linguist-vendored
// jquery.js linguist-vendored=false - What's this for?
`
@Alhadis

**Comment by Alhadis** [comment]
Yes. `linguist-vendored` tells Linguist that the directories or files aren't to be included in a repository's language breakdown.

> `jquery.js linguist-vendored=false` - What's this for?

That does the opposite: it instructs Linguist **not** to exclude files from language analysis. By default, Linguist already excludes certain files and folders that are very commonly used in projects. Examples include jQuery, Bootstrap, React, etc... the full list of files that are [excluded are here](https://github.com/github/linguist/blob/master/lib/linguist/vendor.yml).

In short, it's unlikely you'll ever need to use `linguist-vendored=false`.

**Comment by pchaigno** [comment]
Looks like the question was answered by @Alhadis (Thanks!)

@jamesone Please reopen if this is still an issue.
